google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsConstants.java

```diff
@@ -0,0 +1,165 @@
+import java.util.Map;
+import java.util.Set;
+
+/** Defining Bigtable builit-in metrics scope, attributes, metric names and views. */
+@InternalApi
+public class BuiltinMetricsConstants {
+
+  public static final AttributeKey<String> PROJECT_ID = AttributeKey.stringKey("project_id");
+  public static final AttributeKey<String> INSTANCE_ID = AttributeKey.stringKey("instance");
+  public static final AttributeKey<String> TABLE_ID = AttributeKey.stringKey("table");
+  public static final AttributeKey<String> CLUSTER_ID = AttributeKey.stringKey("cluster");
+  public static final AttributeKey<String> ZONE_ID = AttributeKey.stringKey("zone");
+  static final AttributeKey<String> CLIENT_UID = AttributeKey.stringKey("client_uid");
+
+  public static final AttributeKey<String> APP_PROFILE = AttributeKey.stringKey("app_profile");
+  public static final AttributeKey<Boolean> STREAMING = AttributeKey.booleanKey("streaming");
+  static final AttributeKey<String> METHOD = AttributeKey.stringKey("method");
+  static final AttributeKey<String> STATUS = AttributeKey.stringKey("status");
+  static final AttributeKey<String> CLIENT_NAME = AttributeKey.stringKey("client_name");
+
+  public static final String OPERATION_LATENCIES_NAME = "operation_latencies";
+  static final String ATTEMPT_LATENCIES_NAME = "attempt_latencies";
+  static final String RETRY_COUNT_NAME = "retry_count";
+  static final String CONNECTIVITY_ERROR_COUNT_NAME = "connectivity_error_count";
+  static final String SERVER_LATENCIES_NAME = "server_latencies";
+  static final String FIRST_RESPONSE_LATENCIES_NAME = "first_response_latencies";
+  static final String APPLICATION_BLOCKING_LATENCIES_NAME = "application_latencies";
+  static final String CLIENT_BLOCKING_LATENCIES_NAME = "throttling_latencies";
+
+  private static final Aggregation AGGREGATION_WITH_MILLIS_HISTOGRAM =
+      Aggregation.explicitBucketHistogram(
+          ImmutableList.of(
+              0.0, 0.01, 0.05, 0.1, 0.3, 0.6, 0.8, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 8.0, 10.0, 13.0,
+              16.0, 20.0, 25.0, 30.0, 40.0, 50.0, 65.0, 80.0, 100.0, 130.0, 160.0, 200.0, 250.0,
+              300.0, 400.0, 500.0, 650.0, 800.0, 1000.0, 2000.0, 5000.0, 10000.0, 20000.0, 50000.0,
+              100000.0, 200000.0, 400000.0, 800000.0, 1600000.0, 3200000.0)); // max is 53.3 minutes
+
+  public static final String METER_NAME = "bigtable.googleapis.com/internal/client/";
+
+  static final Set<String> COMMON_ATTRIBUTES =
+      ImmutableSet.of(
+          PROJECT_ID.getKey(),
+          INSTANCE_ID.getKey(),
+          TABLE_ID.getKey(),
+          APP_PROFILE.getKey(),
+          CLUSTER_ID.getKey(),
+          ZONE_ID.getKey(),
+          METHOD.getKey(),
+          CLIENT_NAME.getKey());
+
+  static void defineView(
+      ImmutableMap.Builder<InstrumentSelector, View> viewMap,
+      String id,
+      Aggregation aggregation,
+      InstrumentType type,
+      String unit,
+      Set<String> extraAttributes) {
```

Comment:
Make this is a Set<Attribute> so that you can avoid calling getKey() for every view